docs/internals/contributing/writing-code/pull-requests.txt

```diff
@@ -340,21 +344,23 @@ If the pull request passes all the criteria below and is not your own, please
 set the "Triage Stage" on the corresponding Trac ticket to "Ready for checkin".
 If you've left comments for improvement on the pull request, please tick the
 appropriate flags on the Trac ticket based on the results of your review:
-"Patch needs improvement", "Needs documentation", and/or "Needs tests". As time
-and interest permits, mergers do final reviews of "Ready for checkin" tickets
-and will either commit the changes or bump it back to "Accepted" if further
-work needs to be done.
+"Pull request needs improvement", "Needs documentation", and/or "Needs tests".
+As time and interest permits, mergers do final reviews of "Ready for checkin"
+tickets and will either commit the changes or bump it back to "Accepted" if
+further work needs to be done.
 
 If you're looking to become a member of the `triage & review team
 <https://www.djangoproject.com/foundation/teams/#triage-review-team>`_, doing
 thorough reviews of contributions is a great way to earn trust.
 
-Looking for a patch to review? Check out the "Patches needing review" section
-of the `Django Development Dashboard <https://dashboard.djangoproject.com/>`_.
+Looking for a pull request to review? Check out the "Pull requests needing
+review" section of the `Django Development Dashboard`_.
```

Comment:
Sure you're aware but that needs a change this is the website somewhere

Reply:
Thanks for pointing it out. This action item is tracked in https://github.com/django/djangoproject.com/issues/1724